splitter.py

```diff
@@ -2,6 +2,10 @@
 from sgmllib import SGMLParser
 
 import codecs
+import sys
+import re
+reload(sys)
+sys.setdefaultencoding('utf8')
```

Comment:
Using sys.setdefaultencoding is a bad practice. I think there are better ways you can achieve what you need (e.g. PYTHONIOENCODING environment variable).

Reply:
Thank you and I will try to figure out the better method~